transmission/transmission.go

```diff
@@ -421,8 +422,8 @@ func (b *batchAgg) fireBatch(events []*Event) {
 		return
 	}
 
-	// build the HTTP request
-	url.Path = buildReqestPath(url.Path, dataset)
+	// build the HTTP request URL
+	url := buildReqestPath(apiHost, dataset)
```

Comment:
you didn't make this typo, but since you're touching it, could you please fix it? 
```suggestion
	url := buildRequestPath(apiHost, dataset)
```